test/integration/smoke-testing/test-login-failures.js

```diff
@@ -31,17 +29,17 @@ test('Trying to sign in with no password using scratchr2 navbar', t => {
     var nonsenseusername = Math.random().toString(36)
         .replace(/[^a-z]+/g, '')
         .substr(0, 5);
-    clickText('Sign in')
-        .then(() => findByXpath('//input[@id="login_dropdown_username"]'))
+    clickCss('.dropdown-toggle')
+        .then(() => findByCss('form#login input#login_dropdown_username'))
```

Comment:
If you're eventually targeting an id, you don't have to use anything except for that id.  `#login #login_dropdown_username` should be overly specific since `#login_dropdown_username` alone should identify the thing with that id.  The xpath only used one id, is there a reason you added `form#login`?